config/dispatcher/dispatcher.ex

```diff
@@ -49,11 +37,50 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://resource/measure-concepts/"
   end
 
+  ###############################################################
+  # files
+  ###############################################################
+  get "/files/:id/download", %{ accept: [:any], layer: :api} do
```

Comment:
I'm not sure I understand why the config needs to be so specific here. Any reason why the example from the docs doesn't work?

```ex
  match "/files/*path" do
    Proxy.forward conn, path, "http://file/files/"
  end
  ```

Is it to restrict certain requests in the future or something?

Reply:
the reason is we use two kind of microservices to handle the upload. One for the upload / download (mu-file-service), one for the metadata (mu-cl-resources / ember data). It's basically how it was done in loket